MetaMask/metamask-extension
# Hardware wallet copy updates, referral and tutorial buttons

## Explanation
We need to start adding referral links to our hardware wallet partners, whilst we're doing this we may as well update our tutorial links as well (some of them were super old and broken). I've also included a number of minor copy changes.  


## More Information
I proposed this in Slack and various stakeholders agreed with this approach: https://consensys.slack.com/archives/C026NDN79JB/p1652271544204939 


## Screenshots/Screencaps






### Before

https://user-images.githubusercontent.com/7900773/169054132-25b25a10-cd26-428d-973e-d0b2b1b088cf.mov




### After
https://user-images.githubusercontent.com/7900773/169053052-af616d01-4383-40bd-a057-9a7e47cb09ac.mov


## Manual Testing Steps
Go to the "Connect Hardware Wallet" screen to click each hardware wallet section to see all the copy changes and buttons.


## Pre-Merge Checklist
